4.6.2.16/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:wheezy
+
+MAINTAINER Jo Shields <jo.shields@xamarin.com>
+
+#based on dockerfile by Michael Friis <friism@gmail.com>
+
+ENV MONO_VERSION 4.6.2.16
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
No worries, I almost made a similar mistake a while ago when copying the files for a new release :smile:

We should probably start automating this.